src/archetype/createProject/SpecifyArtifactIdStep.ts

```diff
@@ -13,7 +13,7 @@ export class SpecifyArtifactIdStep implements IProjectCreationStep {
             const inputBox: InputBox = window.createInputBox();
             inputBox.title = "Create Maven Project";
             inputBox.placeholder = "e.g. demo";
-            inputBox.prompt = "Input artifact Id of your project.";
+            inputBox.prompt = "Input artifact Id of your project. (also used as project name)";
```

Comment:
VS Code will also add a tip with brackets, so the prompt looks like: `Input artifact Id of your project. (also used as project name) (Press 'Enter' to confirm or 'Escape' to cancel...)`, which looks wired for double bracket string.

How about this: `Input artifact Id as well as project name of your project.`

Reply:
I would use `Input artifact Id (also as project name) of your project.` WDYT?

Reply:
sounds good.